src/main/java/uk/gov/pay/api/service/telephone/CreateTelephonePaymentService.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.pay.api.service.telephone;
+
+import uk.gov.pay.api.model.telephone.CreateTelephonePaymentRequest;
+import uk.gov.pay.api.model.telephone.PaymentOutcome;
+import uk.gov.pay.api.model.telephone.State;
+import uk.gov.pay.api.model.telephone.Supplemental;
+import uk.gov.pay.api.model.telephone.TelephonePaymentResponse;
+
+public class CreateTelephonePaymentService {
+    
+    public TelephonePaymentResponse create(CreateTelephonePaymentRequest createTelephonePaymentRequest){
+        
+        Supplemental supplemental = createTelephonePaymentRequest.getPaymentOutcome().getSupplemental()
+                .map(s -> new Supplemental(
+                        s.getErrorCode(),
+                        s.getErrorMessage()
+                )).orElse(null);
+        
+        PaymentOutcome paymentOutcome = new PaymentOutcome(
+                createTelephonePaymentRequest.getPaymentOutcome().getStatus(),
+                createTelephonePaymentRequest.getPaymentOutcome().getCode(),
+                supplemental
+        );
+        
```

Comment:
Not sure of the purpose of these lines -  you are basically cloning the incoming payment outcome, but not sure why

Reply:
I was playing around with Optionals when checking whether the JSON returned as expected. Then I realised that these lines were unnecessary so I've removed them.